### PR TITLE
Register presubmit for bridge-marker 0.6

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horáček <phoracek@redhat.com>

### CODE REVIEW

**Review by oshoval — APPROVED** [review]
Thanks

**Comment by oshoval** [comment]
@qinqon can you take a look please on the PR of Petr ?

File: github/ci/prow-deploy/files/jobs/kubevirt/bridge-marker/bridge-marker-presubmits-0.6.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+presubmits:
+  kubevirt/bridge-marker:
+    - name: pull-bridge-marker-test
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/bridge-marker/bridge-marker-presubmits-0.6.yaml:4]: maybe need to add release-0.6 to the name ?

Comment (phoracek) [reply github/ci/prow-deploy/files/jobs/kubevirt/bridge-marker/bridge-marker-presubmits-0.6.yaml]: Sure, done

**Review by oshoval — APPROVED** [review]
Thanks

**Pull request merged**